index.html

```diff
@@ -3,1359 +3,2011 @@
 <head>
   <meta charset="utf-8" />
   <meta property="og:type" content="website">
-  <meta property="og:title" content="Community Projects" />
-  <meta property="og:image" content="https://raw.githubusercontent.com/CyberDiscovery/cyberdiscovery.github.io/master/Image_Assets/CD_LOGO_SQUARE.png" />
-  <meta property="og:site_name" content="Cyber Discovery" />
+  <meta property="og:title" content="Cyber Discovery" />
+  <meta property="og:image" content="static/logo.jpg" />
+  <meta property="og:site_name" content="Discord Community" />
   <meta property="og:type" content="object" />
-  <meta property="og:url" content="https://cyberdiscovery.github.io" />
-  <meta property="og:description" content="A website showcasing the work of the Cyber Discovery community." />
-  <meta name="description" content="A website showcasing the work of the Cyber Discovery community.">
-  <meta name="keywords" content="CyberDiscovery,GitHub,Community,Projects">
+  <meta property="og:url" content="https://cyberdiscoverycommunity.uk" />
+  <meta property="og:description" content="A friendly, growing community of students aged 13-18 taking part in SANS' CyberStart family of programmes." />
+  <meta name="description" content="A friendly, growing community of students aged 13-18 taking part in SANS' CyberStart family of programmes.">
+  <meta name="keywords" content="Cyber Discovery,Discord,Community,GitHub,Projects">
   <meta name="author" content="Sh3llcod3">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <meta name="theme-color" content="#3f51b5" />
+  <meta name="theme-color" content="#1b1b21" />
   <meta name="referrer" content="origin">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-  <link rel="stylesheet" href="https://unpkg.com/material-components-web@3.2.0/dist/material-components-web.min.css">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" type="text/css">
-  <link rel="stylesheet" href="./Site_Code/styles.min.css" type="text/css">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" type="text/css">
-  <script src="https://unpkg.com/material-components-web@3.2.0/dist/material-components-web.min.js"></script>
-  <script src="https://player.vimeo.com/api/player.js"></script>
-  <title>Cyber Discovery Community</title>
-  <link rel="icon" href="./Image_Assets/favicon.png">
-  <style>
-  </style>
+  <link rel="stylesheet" href="dist/bundle.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="static/normalize.min.css" type="text/css">
+  <script src="dist/bundle.js" defer></script>
+  <title>CD Discord Community | Home</title>
```

Comment:
```suggestion
  <title>Cyber Discovery Community | Home</title>
```

Reply:
Same as above, am happy to change this, but it will clip on smaller viewports, if that's cool and good then alright.